### DISCUSSION TITLE
AWS LocalStack

### DISCUSSION THREAD

**Discussion by cyberslot** [crossplane/crossplane#4549] [Q&A] [answered]
Hi all,

I'm curious about potential integration between Crossplane and AWS LocalStack (similar to Terraform and Pulumi). Is there some perspective (intentions or plans) toward such implementation or such initiative would be entirely in the realm of LocalStack team?

**Comment by stevendborrelli** [comment]
Yes, there is an integration. Please see the docs at https://docs.localstack.cloud/user-guide/integrations/crossplane/

**Comment by cyberslot** [comment, marked as answer]
Yes, there is already, because of the topic (https://discuss.localstack.cloud/t/crossplane-integration/507) I've raised on localstack roughly a month and a half ago.
However, thanks anyway for the response!

**Comment by weitzj** [comment]
Link to a blog-post. MIght be helpful https://medium.com/@verajm/crossplane-argo-cd-and-localstack-for-local-testing-17c34109411f
